core/ide/src/main/java/org/alice/ide/IDE.java

```diff
@@ -485,6 +485,7 @@ protected VirtualMachine createVirtualMachineForSceneEditor() {
 
   public final VirtualMachine createRegisteredVirtualMachineForSceneEditor() {
     VirtualMachine vm = this.createVirtualMachineForSceneEditor();
```

Comment:
i feel like the function name createVirtualMachineForSceneEditor would imply that i've created a vm for the scene editor, and i shouldn't need to call a second function to create it even harder for the scene editor?